jep/4/README.adoc

```diff
@@ -44,7 +44,7 @@ endif::[]
 //
 // Uncomment if discussion will occur in forum other than jenkinsci-dev@ mailing list.
 | Discussions-To
-|
+| link:https://groups.google.com/d/topic/jenkinsci-dev/6-1mZoKp4hM/discussion[jenkinsci-dev@ thread]
```

Comment:
@oleg-nenashev This field is only needed if the discussions in general will occur on a mailing list other than `jenkinsci-dev`